config/config.go

```diff
@@ -1257,6 +1260,43 @@ func (cfg *InstrumentationConfig) ValidateBasic() error {
 	return nil
 }
 
+type DBSyncConfig struct {
+	Enable               bool          `mapstructure:"db-sync-enable"`
+	SnapshotInterval     int           `mapstructure:"snapshot-interval"`
+	SnapshotDirectory    string        `mapstructure:"snapshot-directory"`
+	TimeoutInSeconds     int           `mapstructure:"timeout-in-seconds"`
+	NoFileSleepInSeconds int           `mapstructure:"no-file-sleep-in-seconds"`
+	FileWorkerCount      int           `mapstructure:"file-worker-count"`
+	FileWorkerTimeout    int           `mapstructure:"file-worker-timeout"`
+	TrustHeight          int64         `mapstructure:"trust-height"`
+	TrustHash            string        `mapstructure:"trust-hash"`
+	TrustPeriod          time.Duration `mapstructure:"trust-period"`
+}
+
+func DefaultDBSyncConfig() *DBSyncConfig {
+	return &DBSyncConfig{
+		Enable:               false,
+		SnapshotInterval:     0,
+		SnapshotDirectory:    "",
+		TimeoutInSeconds:     0,
+		NoFileSleepInSeconds: 5,
+		FileWorkerCount:      0,
```

Comment:
Can we have better and more common defaults here? I'd suggest let's provide some meaning ful default value for:

- SnapshotInterval
- TimeoutInSeconds
- NoFileSleepInSeconds
- FileWorkerCount
- FileWorkerTimeout
- TrustPeriod

Reply:
updated